rmgpy/rmg/reactors.py

```diff
@@ -463,10 +463,23 @@ def generate_reactor(self, phase_system):
         if self.evap_cond_conditions:
             kLA_kH_evap_cond = rms.kLAkHCondensationEvaporationWithReservoir(domain,self.evap_cond_conditions)
             interfaces.append(kLA_kH_evap_cond)
-            
+
         react = rms.Reactor(domain, y0, (0.0, self.tf), interfaces, p=p)
         return react, domain, interfaces, p
 
+class ConstantTPIdealGasReactor(Reactor):
```

Comment:
I wonder what is the best way to set up different reactors, is it better to make this reactor to support interfaces, or is it better to write new reactors with interfaces

Reply:
When I designed the system I had in mind that each domain+interfaces combination would be its own reactor. I think that keeps the input a bit simpler for the user. However, on the other hand if we start including a lot of these reactors enabling users to specify interfaces within a reactor instead of having multiple reactors might simplify the code a lot long term and make the system more flexible.